app/models/service_template_ansible_playbook.rb

```diff
@@ -32,7 +32,7 @@ def self.create_catalog_item(options, _auth_user)
     transaction do
```

Comment:
@syncrou 
Doing this in a transaction would prevent the other processes (workers) from seeing this queue entry
and the work will never get done